teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(id, age, specialization, name = 'UnKnown', parent_permission: true)
+    super(id, age, name, parent_permission)
```

Comment:
- [Optional] I think when creating a teacher object, the parameter id won't be required if we previously generated random numbers in the parent Person class above :point_up: